Mage.Client/src/main/java/mage/client/deckeditor/DeckEditorPanel.java

```diff
@@ -1142,20 +1110,12 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         btnAddLand.setText("Lands");
         btnAddLand.setIconTextGap(2);
         btnAddLand.setName("btnAddLand"); // NOI18N
-        btnAddLand.addActionListener(new java.awt.event.ActionListener() {
-            public void actionPerformed(java.awt.event.ActionEvent evt) {
-                btnAddLandActionPerformed(evt);
```

Comment:
It's auto-generated code from NetBeans 8, so do not change it -- it will be rewritten on next form save (all *.form files generates code for *.java files, see `//GEN-BEGIN` blocks in code).

So it must be reverted.